api/src/main/resources/liquibase-update-to-2.1.xml

```diff
@@ -266,10 +266,18 @@
 		<comment>Generating UUIDs for all rows in cohort_member table via built in uuid function.</comment>
 		<update tableName="cohort_member" >
 			<column name="uuid" valueNumeric="name-of-uuid-function" />
+			<where>uuid = ''</where>
```

Comment:
I really don't know :) Shouldn't it not have NULL values because nullable="false"?